### DISCUSSION TITLE
New Lens made with Lens Studio 4.49 not working

### DISCUSSION THREAD

**Discussion by fpgalex** [ptrumpis/snap-camera-server#46] [Q&A] [answered]
Hi!

I've made a new lens using Lens Studio 4.49: https://www.snapchat.com/lens/b01c0176e43449e98328196d367cade3

I can find the lens and it works in Snapchat but not in Snap Camera.  I've done the following to try and resolve the issue:

1. Server has been installed via the AutoConfig file as of July 30th (today).  
2. I've run the 1.21 Signature Patch (https://ptrumpis.github.io/snap-camera-signature-patch/)
3. I've updated Snap Camera using the unofficial update method: https://github.com/ptrumpis/snap-camera-server/wiki/Updating-Snap-Camera-(Unofficial)

A snippet from the AppData\Local\Snap\Snap Camera\Log.txt:

VCam::ActionType::SetActiveLensId QVariant(VCam::LensActivationEvent, )
VCam::ActionType::ActiveLensChanged QVariant(VCam::LensInfo, )
VCam::ActionType::SetCoreProcessingLens QVariant(VCam::CoreProcessingLens, )
VCam::ActionType::SetMainLensHint QVariant(QString, "")
VCam::ActionType::LensDownloaded QVariant(QString, "62809100879")
VCam::ActionType::SetCoreProcessingOverlay QVariant(QImage, QImage(null))
Core: CORE: using complex effect with id lns://C:/Users/arod1/AppData/Local/Snap/Snap Camera/cache/lenses/62809100879/lens.lns@@
Core: [LS::Graphics::Context] this - 00000081A70FB8C0, glContext - 0000000000030001, glContextMode - 2
Core: [SGRenderer] Experiment value: 0
Core: bindScript executed
Core: [engine] engineComplexEffect started loading lens with id 62809100879
Core: [engine] The lens under development mode is OFF
Core: [OpenGL] contextsMap doesn't contain eglContext 0000000000030001
Core: [LS::Graphics::~Context] this - 00000081A70FB8C0, glContext - 0000000000030001, glContextMode - 2
Core: [LS::Graphics::Context] this - 00000081A70FB640, glContext - 0000000000030001, glContextMode - 2
Core: Unknown blend mode 
Core: [engine] engine exception while processing lens with id 62809100879
Core: [OpenGL] contextsMap doesn't contain eglContext 0000000000030001
Core: [LS::Graphics::~Context] this - 00000081A70FB640, glContext - 0000000000030001, glContextMode - 2
"Lens not loaded exception: Unknown blend mode"

Any help is appreciated!  It seems like a incompatibility issue?

**Comment by ptrumpis** [comment, marked as answer]
Based on your shared log data, I can't tell you where the error is. But if you get the chance to release the lens with Lens Studio 4.36 then it should work.

So if downgrading Lens Studio is an option to you it should work.

**Reply by fpgalex** [reply]
Hi Patrick,

Thanks for the response!  This was successful!  I followed the same exact process I did in 4.36 as I did in 4.49.  One hiccup I ran into was the older version does not support .tga textures, only .png/.jpg.  This wasn't a problem since I converted from .tga to .png and issue was easily resolved.
